.rubocop.yml

```diff
@@ -55,3 +52,18 @@ Style/StringLiterals:
 
 Style/WordArray:
   EnforcedStyle: brackets
+
+Style/StringLiterals:
+  Enabled: false
+
+Metrics/ClassLength:
+  Enabled: false
+
+Metrics/ModuleLength:
+  Enabled: false
+
+Metrics/MethodLength:
```

Comment:
There's `DisabledByDefault: true` above, is that enough for these `Length` cops? (It's ok with me to remove the `ClassLength` cop higher up in the file, too)